sql/src/test/java/io/crate/integrationtests/ResizeShardsITest.java

```diff
@@ -60,24 +60,20 @@ public void testShrinkShardsEnsureLeftOverIndicesAreRemoved() throws Exception {
                 "clustered into 3 shards");
         ensureYellow();
 
-        final String targetIndexName = ".resized." + getFqn("quotes");
-        final String backupIndexName = ".backup." + getFqn("quotes");
-        logger.info("targetIndexName:" + targetIndexName );
-
-        createIndex(targetIndexName, backupIndexName);
+        final String resizeIndex = ".resized." + getFqn("quotes");
+        createIndex(resizeIndex);
+        logger.info("targetIndexName:" + resizeIndex );
```

Comment:
could you please remove that? 

I forgot to do so when I fixed the test to work for randomised schema. 
thanks